src/js_app.hpp

```diff
@@ -240,11 +240,9 @@ void AppClass<T>::constructor(ContextType ctx, ObjectType this_object, Arguments
     config.transport = AppClass<T>::transport_generator(Protected(Context::get_global_context(ctx)),
                                                         NetworkTransport::make_dispatcher());
 
-    config.device_info.platform = platform;
```

Comment:
I talked with @RedBeard0531 about this in person (I don't think he made a PR for `main` yet - or at least I can't find it). I think it's annoying that the code now has two ways of determining the platform, one by the SDK which is relied on from the `get_user_agent` function above and one which is provided by core. Ideally Core should expose a way to get the `platform` which could be called from `get_user_agent` or it would simply implement `get_user_agent` itself.